### PR TITLE
Remove the (hidden) hide metadata panel feature.

### LINKED ISSUE #641: Remove ability to hide the metadata panel
PURL is the only app that makes use of this option and with the addition of the IIIF drag and drop icon it might make sense to remove the ability to hide it (which will turn the feature on in PURL).

_See [related conversation](https://github.com/sul-dlss/sul-embed/pull/566#issuecomment-239874495)_

### PR DESCRIPTION
**Pull request by jkeck**
Closes #641

### CODE REVIEW

File: spec/lib/embed/embed_this_panel_spec.rb [hunk 1]
```diff
@@ -100,11 +99,6 @@
         expect(src).to match(/&hide_embed=true/)
         expect(src).to match(/&hide_title=true/)
       end
-
-      it 'does not include parameters that are not intended to be passed' do
```
Comment (ndushay) [inline spec/lib/embed/embed_this_panel_spec.rb]: is this check no longer useful, or is "hide_metadata" no longer a good test string?

File: spec/lib/embed/request_spec.rb [hunk 2]
```diff
@@ -45,17 +37,13 @@
...
-      Embed::Request.new(url: purl, hide_title: 'true', hide_metadata: 'true').as_url_params
+      Embed::Request.new(url: purl, hide_title: 'true').as_url_params
     end
 
     it 'is a hash of parameters to be turned into url optinos' do
       expect(url_params).to be_a Hash
       expect(url_params[:hide_title]).to eq('true')
     end
-
-    it 'does not include parameters that should not be passed along in a url (such as hide_metadata)' do
```
Comment (ndushay) [inline spec/lib/embed/request_spec.rb]: as above, is this check no longer useful, or is "hide_metadata" no longer a good test string?

**Comment by ndushay** [comment]
question about 2 tests;  o.w. LGTM.

File: spec/lib/embed/request_spec.rb (hunk 2 above)
Comment (jkeck) [inline spec/lib/embed/request_spec.rb]: 👍 I'll add a test back in.  This is whitelisting so testing that other things aren't making their way in is a good idea.  Good catch.

**Comment by jkeck** [comment]
Thanks for the feedback @ndushay. I believe I have resolved your concern.

**Comment by ndushay** [comment]
👍 for merging

**Pull request merged**